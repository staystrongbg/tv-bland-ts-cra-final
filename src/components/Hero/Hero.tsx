import styles from '../../styles/Hero.module.scss';
import Popcorn from '../../images/pexels-tima-miroshnichenko-7991378.jpg';

const Hero: React.FC = () => {
  return (
    <section className={`${styles.hero} flex-center`}>
      <div className={styles['img-container']}>
        <img src={Popcorn} alt='' />
      </div>
      <div className={styles.nowPlaying}>
        <p data-testid='hero'>NOW PLAYING</p>
        <h1>
          <span>NOW</span> PLAYING
        </h1>
        <div className={styles.heroDesc}>
          <p>
            TV Shows and web series database
            <br />
            Create personalized schedules. Episode guide cast, crew and
            character information
          </p>
        </div>
      </div>
      <div className='gradient' />
    </section>
  );
};

export default Hero;
