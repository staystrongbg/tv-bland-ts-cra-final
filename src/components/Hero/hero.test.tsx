import React from 'react';
import renderer from 'react-test-renderer';
import Hero from './Hero';

it('render app', () => {
  const render = renderer.create(<Hero />).toJSON();
  expect(render).toMatchSnapshot();
});
