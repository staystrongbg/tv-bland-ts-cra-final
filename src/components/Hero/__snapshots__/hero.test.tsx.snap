// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<section
  className="hero flex-center"
>
  <div
    className="img-container"
  >
    <img
      alt=""
      src="pexels-tima-miroshnichenko-7991378.jpg"
    />
  </div>
  <div
    className="nowPlaying"
  >
    <p
      data-testid="hero"
    >
      NOW PLAYING
    </p>
    <h1>
      <span>
        NOW
      </span>
       PLAYING
    </h1>
    <div
      className="heroDesc"
    >
      <p>
        TV Shows and web series database
        <br />
        Create personalized schedules. Episode guide cast, crew and character information
      </p>
    </div>
  </div>
  <div
    className="gradient"
  />
</section>
`;
